net/mosquitto/Makefile

```diff
@@ -144,7 +144,7 @@ endef
 define Package/libmosquittopp
     SECTION:=libs
     CATEGORY:=Libraries
-    DEPENDS:=+libmosquitto-ssl +libstdcpp
+    DEPENDS:=+libmosquitto +libmosquitto-ssl +libstdcpp
```

Comment:
yes, actually the mosquitto will not get compiled with the latest git version, it is broken because libwebsockets is not compiling because of libwebsockets / cyassl (would be nice to bumb libwebsockets to the latest version or have mosquitto package without libwebsockets)